lib/blueprinter/view.rb

```diff
@@ -10,6 +10,20 @@ def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [
       @excluded_field_names = excluded_view_names
     end
 
+    def inherit(view)
+      view.fields.values.each do |field|
+        self << field
```

Comment:
I'll defer to @philipqnguyen for desired behavior, but due to the implementation of `<<` in this class, we'd raise an error if a field name collides, however, in inheritance -- you'd expect those fields to be overridable.

Reply:
I agree, would be nice to have some specs to ensure that the last defined field is the field implementation that gets set. Both in a single view with multiple duplicate field name implementations, as well for the case where multiple classes define a field, to ensure the lowest subclass implementation gets set.

Reply:
Nice catch. I'll add the specs and remove the guard, then.

Reply:
I added a spec for the subclassing scenario and modified the existing spec for the regular overriding scenario. Let me know what you think